packages/salesforcedx-vscode-core/src/util/authInfo.ts

```diff
@@ -114,6 +114,15 @@ export class OrgAuthInfo {
     );
   }
 
+  public static async isAProductionOrg(username: string): Promise<boolean> {
```

Comment:
Please review https://forcedotcom.github.io/sfdx-core/classes/org_org.Org-1.html to determine this.

Reply:
@peternhale is there an affirmative case for "this is a production org"? I'm not seeing that in these docs, so it's looking like Production would be the fallback case.

Reply:
@randi274 Yes. production should be the fallback.

Reply:
Got it! @interfaceconjurer and I were chatting about this detail today, it might be worth y'all discussing the possibility of "Production" being overly inclusive.